vulnerabilities/helpers.py

```diff
@@ -164,3 +165,34 @@ def __lt__(self, other):
         )
 
     return affected_package_with_patched_package_objects
+
+
+def split_markdown_front_matter(lines: str) -> Tuple[str, str]:
```

Comment:
I would call this `text` rather than `lines` which suggests we have a list of lines and we have not.

Reply:
fixed